config/locales/help_text/en.yml

```diff
@@ -6,11 +6,9 @@ en:
       password and receive a security code on your phone.
     requested_attributes:
       intro_html: This is the only information %{app_name} will share with %{sp}
-      address: Mailing address
+      address1: Mailing address
       birthdate: Date of birth
       email: Email address
-      family_name: Last name
-      given_name: First name
-      name: Full name
+      family_name: Full name
```

Comment:
I think we should have an explicit mapping in code from `given_name` and `family_name` to "Full name" -- and the same with the address attributes. Otherwise if an SP for some reason asks only for first name or just city, we still display the right thing

We could do something like:

```ruby
SORTED_ATTRIBUTE_MAPPING = [
  [[:given_name, :family_name], :full_name],
  [[:address1, :address1, :city, :state, :zip], :mailing_address],
  # ...
]

requested_sorted_attributes = SORTED_ATTRIBUTE_MAPPING.map do |raw_attributes, display_attribute|
  if (requested_attributes & raw_attributes).present?
    display_attribute
  end
end.compact

```